lib/chef/application/solo.rb

```diff
@@ -21,6 +21,7 @@
 require_relative "client"
 require "fileutils" unless defined?(FileUtils)
 require "pathname" unless defined?(Pathname)
+require "chef-utils"
```

Comment:
Should I change this to the following?
```suggestion
require "chef-utils/dsl/windows"
require "chef-utils/dist"
```